src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -110,14 +116,68 @@ std::vector<image_annotation> parse_annotations(
     // instances we report will actually equal the number of image_annotation
     // values.
     if (has_label && has_x && has_y && annotation.bounding_box.area() > 0.f) {
+      
+      // If the annotations are normalised, they will range between 0 and 1  
+      if (annotation_scale_ == annotation_scale_enum::NORMALIZED) {
+        image_height = 1;
```

Comment:
Upon further thought, I think this would be clearest if we defined new local variables, e.g.
```
float annotation_image_height;
float annotation_image_width;
switch (annotation_scale_) {
case annotation_scale_enum::PIXEL:
  annotation_image_height = static_cast<float>(image_height);
  annotation_image_width = static_cast<float>(image_width);
case annotation_scale_enum::NORMALIZED:
  annotation_image_height = 1.f;
  annotation_image_width = 1.f;
  break;
}
```